File: 7_kyu/Printer_Errors.kt
// Returns errors statistic of s "errors/totals" (decimal numbers)
// Errors are all letters not in range 'a'..'m'.
fun printerError(s: String): String {
  val total = s.length
  val errors = s.count {it !in 'a'..'m'}
  return "$errors/$total"
}

File: 7_kyu/Printer_Errors.r
# Returns summary "#erros/#all".
# where #all is the number of all messages (characters) in s
# and #errors are the number of messages, no a,b,c...,m.
# Numbers are decimal numbers in the returned string.
printerError <- function(s) {
  all <- nchar(s)
  errors <- nchar(gsub("[a-m]","",s))
  paste(errors,all,sep="/")
}
